aws-lambda/src/http.rs

```diff
@@ -0,0 +1,56 @@
+use std::sync::Arc;
+
+use anyhow::Result;
+use hyper::body::Bytes;
+use reqwest::Client;
+use tailcall::http::Response;
+use tailcall::HttpIO;
+
+#[derive(Clone)]
+pub struct LambdaHttp {
+    client: Client,
+}
+
+impl Default for LambdaHttp {
+    fn default() -> Self {
+        Self { client: Client::new() }
+    }
+}
+
+impl LambdaHttp {
+    pub fn init() -> Self {
+        Default::default()
+    }
+}
+
+#[async_trait::async_trait]
+impl HttpIO for LambdaHttp {
+    async fn execute(&self, request: reqwest::Request) -> Result<Response<Bytes>> {
+        let client = self.client.clone();
+        let method = request.method().clone();
+        let url = request.url().clone();
+        let response = client.execute(request).await?.error_for_status()?;
+        let res = Response::from_reqwest(response).await?;
+        tracing::info!("{} {} {}", method, url, res.status.as_u16());
+        Ok(res)
+    }
+}
+
+pub fn to_response(
+    response: hyper::Response<hyper::Body>,
+) -> anyhow::Result<lambda_http::Response<hyper::Body>> {
+    Ok(response)
+}
```

Comment:
its usage unwraps and wraps its to the result again `Ok::<Response<hyper::Body>, Error>(to_response(resp)?)` so one of the part could be simplified. This function is probably redundant